### ISSUE TITLE
aria-label should be set to Remove only in removable state of QChip 

### ISSUE THREAD

**Issue by dwaxweiler** [quasarframework/quasar#14561]
### What happened?

Since #14388 has been merged for version 2.8.4, the `aria-label` is set to `Remove` by default on `clickable` state of QChip. Alternatively `remove-aria-label` can be set to have a different text. The `clickable` state is not equal to the `removable` state.

### What did you expect to happen?

The `aria-label` should not be set to `Remove` or to the value of `remove-aria-label` on QChip when it is in `clickable` state. It is fine for `removable` state though.

### Reproduction URL

https://jsfiddle.net/de1zgk67/

### How to reproduce?

1. Go to the provided reproduction link.
2. Inspect the QChip element in DevTools
3. Notice that the `div` tag with the class `q-chip` has `aria-label="Remove"`.

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

**Comment by pdanpdan** [comment]
Will be fixed in next version of Qv1 and Qv2
